cms/templates/js/mock/mock-settings-page.underscore

```diff
@@ -62,6 +62,21 @@
         <span class="tip tip-stacked timezone">(UTC)</span>
       </div>
     </li>
+    <li>
+        <label id="heading-entrance-exam">${_("Entrance Exam")}</label>
+            <div class="show-data">
+                <div class="heading">
+                    <input type="checkbox" id="entrance-exam-enabled" />
+                    <label for="entrance-exam-enabled">${_("Enables entrance exam requirement. Students must successfully complete exam before gaining access to the course.")}</label>
```

Comment:
Text for this check box:

Require students to pass an exam before beginning the course.